.github/workflows/build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           node-version: '18'
 
       - name: Install dependencies
-        run: npm install
+        run: npm install --legacy-peer-deps
```

Comment:
We don't need that. @muhammad-bilal-ali-khokhar didn't we decide to remove this?